src/callAPIMethod.js

```diff
@@ -50,6 +54,28 @@ export default function callAPI(
             .then( result => {
                 if (!response.ok) throw new APIError(response.status, response.statusText, result);
 
+                if (responseOptions.fullResponse) {
+                    // https://developer.mozilla.org/en-US/docs/Web/API/Response
+                    const writableResponse = [
+                        'headers',
+                        'ok',
+                        'redirected',
+                        'status',
+                        'statusText',
+                        'type',
+                        'url',
+                        'useFinalURL',
+                        'bodyUsed',
+                        'body'
+                    ].reduce((res, key) => {
+                        res[key] = response[key];
+
+                        return res;
+                    }, {});
+
+                    return Object.assign({}, writableResponse, { body: result });
```

Comment:
What about that?
```
return {...response, body: result};
```

Reply:
Response don't have enumerable properties. It wouldn't work, but rewrite it to `{ ...writableResponse, ...{ body: result }` is nice idea.